src/lib/statistics/column_statistics.cpp

```diff
@@ -57,7 +57,7 @@ FilterByValueEstimate ColumnStatistics<ColumnDataType>::estimate_predicate_with_
       // distinction between integers and floats
       // for integers "< value" means that the new max is value <= value - 1
       // for floats "< value" means that the new max is value <= value - ε
-      if (std::is_integral_v<ColumnDataType>) {
+      if constexpr (std::is_integral_v<ColumnDataType>) {
```

Comment:
You'll probably get a false positive error with clang-tidy here, at least until we upgrade to LLVM 7. Use NOLINT if necessary

Reply:
Neat, I did not know clang-tidy supports NOLINT as well.